### PR TITLE
Lesson-3 HW

### PR DESCRIPTION
**Pull request by anastasiya-harbatsevich**
p1. Calculator
p2. MinMax
p3. Converter
p4. Division
p5. Temperature

### CODE REVIEW

File: src/main/java/org/example/homework3/Calculator.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        System.out.println("Subtraction is: " + Subtraction);
+
+        //Multiplication
+        int multiplication = a * b;
+        System.out.println("Multiplication is: " + multiplication);
+
+        //Division
+        double division = (double) a / b;
+        System.out.println("Division is: " + division);
+
```
Comment (alexshamrai) [inline src/main/java/org/example/homework3/Calculator.java]: Extra line, please remove

File: src/main/java/org/example/homework3/Calculator.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package org.example.homework3;
+public class Calculator {
+
+    public static void main(String[] args) {
+        int a = 5;
+        int b = 9;
+
+        //Sum
```
Comment (alexshamrai) [inline src/main/java/org/example/homework3/Calculator.java]: I guess those comments are not needed. The naming of variables is pretty clear, what is the purpose of that comment? You can leave it as it is, no problem. But comments are needed only if the code is not obvious and straightforward

File: src/main/java/org/example/homework3/Converter.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package org.example.homework3;
+
+public class Converter {
+    public static void main(String[] args) {
+
+        long a = 5675876500000000L;
+        int b = (int) a;
+        System.out.println("Original is : " + a);
+        System.out.println("Converted is : " + b);
+
```
Comment (alexshamrai) [inline src/main/java/org/example/homework3/Converter.java]: extra line, please remove

File: src/main/java/org/example/homework3/Division.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+    public static void main(String[] args) {
+
+        int a = 10;
+        int b = 4;
+        int integer = a / b;
+        int remainder = a % b;
+        System.out.println("Integer is: " + integer);
+        System.out.println("Remainder is: " + remainder);
+
```
Comment (alexshamrai) [inline src/main/java/org/example/homework3/Division.java]: extra line, please remove

File: src/main/java/org/example/homework3/Division.java [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    }
+
```
Comment (alexshamrai) [inline src/main/java/org/example/homework3/Division.java]: extra line, please remove

File: src/main/java/org/example/homework3/MinMax.java [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+        int a = 10;
+        int b = 4;
+
+        //Min
+        System.out.println(Math.min(a, b));
+
+        //Max
+        System.out.println(Math.max(a, b));
+    }
+
```
Comment (alexshamrai) [inline src/main/java/org/example/homework3/MinMax.java]: extra line, please remove

File: src/main/java/org/example/homework3/Temperature.java [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+public class Temperature {
+
+    public static void main(String[] args) {
+        float celsius = 23;
+        float fahrenheit = celsius * 9 / 5 + 32;
+        double kelvin = celsius + 273.15;
+        System.out.println("Temperature in fahrenheit is: " + fahrenheit);
+        System.out.println("Temperature in kelvin is: " + kelvin);
+    }
+
```
Comment (alexshamrai) [inline src/main/java/org/example/homework3/Temperature.java]: extra line, please remove

**Review by alexshamrai — APPROVED** [review]
Everything is good, except minor comments related to style

File: src/main/java/org/example/homework3/Calculator.java (hunk 1 above)
Comment (anastasiya-harbatsevich) [reply src/main/java/org/example/homework3/Calculator.java]: Removed

File: src/main/java/org/example/homework3/Calculator.java (hunk 2 above)
Comment (anastasiya-harbatsevich) [reply src/main/java/org/example/homework3/Calculator.java]: Removed

File: src/main/java/org/example/homework3/Converter.java (hunk 3 above)
Comment (anastasiya-harbatsevich) [reply src/main/java/org/example/homework3/Converter.java]: Removed

File: src/main/java/org/example/homework3/Division.java (hunk 4 above)
Comment (anastasiya-harbatsevich) [reply src/main/java/org/example/homework3/Division.java]: Removed

File: src/main/java/org/example/homework3/Division.java (hunk 5 above)
Comment (anastasiya-harbatsevich) [reply src/main/java/org/example/homework3/Division.java]: Removed

File: src/main/java/org/example/homework3/MinMax.java (hunk 6 above)
Comment (anastasiya-harbatsevich) [reply src/main/java/org/example/homework3/MinMax.java]: Removed

File: src/main/java/org/example/homework3/Temperature.java (hunk 7 above)
Comment (anastasiya-harbatsevich) [reply src/main/java/org/example/homework3/Temperature.java]: Removed

**Review by alexshamrai — APPROVED** [review]
Awesome, thanks

**Pull request merged**